geometry_msgs/PointStamped point_stamped
std_msgs/ColorRGBA color_rgba
